packages/admin-ui/src/filters/filter-base.tsx

```diff
@@ -17,35 +16,26 @@ import { Stack } from '../stack'
 import * as style from './filter.style'
 
 export function BaseFilter(props: BaseFilterProps) {
-  const { state, children, appliedValuesLabel = '' } = props
-  const { onClear, onChange, label, menu, combobox, baseId } = state
+  const { state, children } = props
+  const { onClear, onChange, menu, combobox, baseId } = state
 
   const formatMessage = useMessageFormatter(messages.actions)
 
   const optionsContainerRef = useRef<HTMLDivElement>(null)
 
   const popoverId = baseId ? `${baseId}-popover` : undefined
-  const disclosureId = baseId ? `${baseId}-disclosure` : undefined
 
-  // TODO investigate arikit buggy typing
-  const comboboxListId = {
-    id: baseId ? `${baseId}-combobox-list` : undefined,
-  } as any
+  const comboboxListId = { id: baseId ? `${baseId}-combobox-list` : undefined }
 
   const scrollHeight = optionsContainerRef?.current?.scrollHeight ?? 0
   const containerHeight = optionsContainerRef?.current?.clientHeight ?? 0
 
   return (
     <>
-      <FilterDisclosure state={menu} id={disclosureId}>
-        {label}
-        {appliedValuesLabel}
-      </FilterDisclosure>
-
       <FilterPopover state={menu} id={popoverId}>
         <tag.span ref={optionsContainerRef} csx={style.scrollableContainer}>
           <Stack
-            as={ComboboxList}
+            as={ComboboxList as any}
```

Comment:
Why not use the ComboboxList component with the Stack inside of it?

Reply:
Not sure if it would break something acessibility-wise. Considering the list should have specific props if it's a combobox-list.

Either way I want to think about a better way to use these components other than "as" but I won't do that on this pr

Reply:
ok 😉